sites/website/sidebars.js

```diff
@@ -21,32 +21,35 @@ module.exports = {
                     items: [
                         "components/accordion",
                         "components/anchor",
+                        "components/anchored-region",
+                        "components/avatar",
                         "components/badge",
                         "components/breadcrumb",
-                        "components/breadcrumb-item",
                         "components/button",
                         "components/card",
                         "components/checkbox",
+                        "components/combobox",
                         "components/data-grid",
                         "components/dialog",
+                        "components/disclosure",
                         "components/divider",
                         "components/flipper",
+                        "components/horizontal-scroll",
                         "components/listbox",
                         "components/listbox-option",
                         "components/menu",
-                        "components/menu-item",
                         "components/number-field",
                         "components/progress",
                         "components/radio",
                         "components/radio-group",
```

Comment:
In a similar vein to these other changes where we are consolidating some components that need to be explained in the context of a wrapping component, should radio button exist inside radio-group?

Reply:
Radios don't have to be inside a group so they are broken out. But the menu item can really only be inside of a menu. So, those are combined into the same doc. It was really strange having it on its own I think.

Reply:
@EisenbergEffect with ARIA radios this is actually not the case - the radios definitely do need to be a child of an element with a role of radiogroup. Radio's which use the native input w/ type="radio" can be linked together with special magic, but from an ARIA standpoint that's not possible or valid; these are only tied together under the same parent. There is ongoing conversations in Open UI around this, but it would require changes from the ARIA working group and likely whatwg.

Reply:
But they don't need to be the child of our radio-group component specifically, correct? I can update the radio example. This would also mean that the example that we've had on our site for many months now is incorrect...

Reply:
Improved the examples for radio and radio-group.